src/test/regress/expected/drop_partitioned_table.out

```diff
@@ -0,0 +1,469 @@
+--
+-- DROP_PARTITIONED_TABLE
+--
+-- Tests to make sure that we properly drop distributed partitioned tables
+--
+SET citus.next_shard_id TO 720000;
+SET citus.shard_count TO 4;
+SET citus.shard_replication_factor TO 1;
+CREATE SCHEMA drop_partitioned_table;
+SET search_path = drop_partitioned_table;
+-- create a function that allows us to see the values of
+-- original and normal values for each dropped table
+-- coming from pg_event_trigger_dropped_objects()
+-- for now the only case where we can distinguish a
+-- dropped partition because of its dropped parent
+-- is when both values are false: check citus_drop_trigger
+CREATE FUNCTION check_original_normal_values()
+    RETURNS event_trigger LANGUAGE plpgsql AS $$
+DECLARE
+    v_obj record;
+BEGIN
+    FOR v_obj IN SELECT * FROM pg_event_trigger_dropped_objects()
+        WHERE object_type IN ('table', 'foreign table')
+    LOOP
+	    RAISE NOTICE 'dropped object: %.% original: % normal: %',
+            v_obj.schema_name,
+            v_obj.object_name,
+            v_obj.original,
+            v_obj.normal;
+    END LOOP;
+END;
+$$;
+CREATE EVENT TRIGGER new_trigger_for_drops
+   ON sql_drop
+   EXECUTE FUNCTION check_original_normal_values();
+-- CASE 1
+-- Dropping the parent table
+CREATE TABLE parent (x text, t timestamptz DEFAULT now()) PARTITION BY RANGE (t);
+CREATE TABLE child1 (x text, t timestamptz DEFAULT now());
+ALTER TABLE parent ATTACH PARTITION child1 FOR VALUES FROM ('2021-05-31') TO ('2021-06-01');
+CREATE TABLE child2 (x text, t timestamptz DEFAULT now());
+ALTER TABLE parent ATTACH PARTITION child2 FOR VALUES FROM ('2021-06-30') TO ('2021-07-01');
+SELECT create_distributed_table('parent','x');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+\d
```

Comment:
minor: `\d` may change output between PG versions, and I'm not sure if the order is guaranteed to be consistent. Can we maybe select ... from pg_class .. ORDER BY ..